components/eamxx/docs/index.md

```diff
@@ -1,10 +1,11 @@
-# The C++ E3SM Atmosphere Model (EAMxx)
+# The E3SM Atmosphere Model in C++ (EAMxx)
 
-Some nice introductory text goes here! Maybe some figures, too. Who knows?!
+EAMxx is an entirely new atmosphere model for E3SM. It is written in C++ using the Kokkos performance portability library to enable it to run efficiently on CPUs, GPUs, and (hopefully) whatever comes next. Currently only the km-scale explicit-convection version called SCREAM (the Simple Cloud-Resolving E3SM Atmosphere Model) is available, but a low-resolution version is in the works.
```

Comment:
Phrases like "entirely new" seem destined to go out of date by not being updated when we start to feel like the model is "not new" anymore, so I would use a tone more apt for documentation rather than an advertisement.

Reply:
I meant "entirely new" in the sense that it doesn't share any parameterizations with EAMf90... though I suspect that will change with some of the low-res parameterizations. How about calling it a "clean-slate atmosphere model"?

Reply:
ok, so we just want to convey that it's not an iterative evolution of EAM.

Here's an idea for a rewrite:

```
EAMxx is one of the available atmospheric models for E3SM that is being built from the ground up using C++ in order to embrace modern software practices and allow "performance portability" across various super computers using the Kokkos library for optimal efficiency on CPUs, GPUs, and whatever comes next.
```

Reply:
Ok, new version (uploaded soon) will be:
>EAMxx was built from the ground up using C++ in order to embrace modern software practices and to allow "performance portability" across various supercomputers. The latter goal is achieved by using the Kokkos library. EAMxx is a "clean-start" model with almost no similarity to the E3SM atmosphere model used in versions 1-3.  Currently only the km-scale explicit-convection version called SCREAM (the Simple Cloud-Resolving E3SM Atmosphere Model) is available, but a low-resolution version is in the works.